### ISSUE TITLE
Language translation files

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART-ctlscripts#1]
**[Original report](https://bitbucket.org/agriggio/art-ctlscripts/issues/1) by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

This issue should stay open to collect language translation files from contributors.

To submit a new translation, simply add a comment with an attached file.

**Comment by agriggio** [comment]
**Original comment by Paul Matthijsse (Bitbucket: [Paul Matthijsse](https://bitbucket.org/Paul Matthijsse), ).**

----------------------------------------

      <div class="preview-container wiki-content"></div>
      <div class="mask"></div>
    </div>
  </div>
</form>

**Comment by agriggio** [comment]
**Original comment by Ohnishi, Yasuo (Bitbucket: [Ohnishi, Yasuo](https://bitbucket.org/Ohnishi, Yasuo), ).**

----------------------------------------

Japanese translation file.

**Comment by agriggio** [comment]
**Original comment by Ohnishi, Yasuo (Bitbucket: [Ohnishi, Yasuo](https://bitbucket.org/Ohnishi, Yasuo), ).**

----------------------------------------

Korean translation file.

**Comment by agriggio** [comment]
**Original comment by Ohnishi, Yasuo (Bitbucket: [Ohnishi, Yasuo](https://bitbucket.org/Ohnishi, Yasuo), ).**

----------------------------------------

Sorry, I uploaded the wrong file. This is the correct Korean file.

**Comment by agriggio** [comment]
**Original comment by Péter Bereczky (Bitbucket: [bykyny](https://bitbucket.org/bykyny), ).**

----------------------------------------

Hungarian translation file. Minor fixes.

**Comment by agriggio** [comment]
**Original comment by Sebastien Guyader (Bitbucket: [Sebastien Guyader](https://bitbucket.org/Sebastien Guyader), ).**

----------------------------------------

**Comment by agriggio** [comment]
**Original comment by Alvaro (Bitbucket: [Alvaro](https://bitbucket.org/Alvaro), GitHub: [Alvaro](https://github.com/Alvaro)).**

----------------------------------------

Italian translation

**Comment by agriggio** [comment]
**Original comment by Péter Bereczky (Bitbucket: [bykyny](https://bitbucket.org/bykyny), ).**

----------------------------------------

**Comment by pbereczky** [comment]
Hungarian translation file. Minor fixes.

[Magyar.txt](https://github.com/user-attachments/files/18553708/Magyar.txt)

**Comment by pbereczky** [comment]
Hungarian translation file.

[Magyar.txt](https://github.com/user-attachments/files/19082053/Magyar.txt)

**Comment by pbereczky** [comment]
Hungarian translation file new version.

[Magyar.txt](https://github.com/user-attachments/files/19141808/Magyar.txt)

**Comment by pbereczky** [comment]
Hungarian translation file. Please don't forget to copy.

[Magyar.txt](https://github.com/user-attachments/files/19339503/Magyar.txt)

**Comment by agriggio** [comment]
Hi @pbereczky, this seems to be for the CTL scripts...

**Comment by pbereczky** [comment]
Yes, it really is. Please overwrite the file in the languages folder with this.

**Comment by agriggio** [comment]
Ok, but this is not the place for submitting such file then. Next time please upload at https://github.com/artpixls/ART-ctlscripts/issues/1

Thanks!

**Comment by pbereczky** [comment]
Thanks!

**Comment by agriggio** [comment]
I actually just realised that I was looking at the wrong place :-( I thought this was the issue in the ART repository! Sorry for the confusion...

**Comment by pbereczky** [comment]
That's all right :-)

**Comment by pbereczky** [comment]
Improved Hungarian language file.

[Magyar.txt](https://github.com/user-attachments/files/20686844/Magyar.txt)

**Comment by pbereczky** [comment]
Added some files related to the current state of ART-ctlscripts:

default file,
Hungarian language file,
sometimes script with magic strings.

Default file and Magyar file cannot have extensions, the .txt extension of scripts needs to be changed to .ctl.

[default.txt](https://github.com/user-attachments/files/24149113/default.txt)

[Magyar.txt](https://github.com/user-attachments/files/24149114/Magyar.txt)

[log.txt](https://github.com/user-attachments/files/24149127/log.txt)

[odt.txt](https://github.com/user-attachments/files/24149128/odt.txt)

[OpenDRT.txt](https://github.com/user-attachments/files/24149131/OpenDRT.txt)

[totm.txt](https://github.com/user-attachments/files/24149134/totm.txt)

[vibrance.txt](https://github.com/user-attachments/files/24149136/vibrance.txt)

**Comment by pbereczky** [comment]
I forgot that the new keys would be at the end of the default file. The first new key is WHITE_RELATIVE_EXPOSURE.

**Comment by pbereczky** [comment]
Hi, Alberto

The OpenDRT-full.ctl script contains the string “Limit LowLimit Low R”. Are you sure this is good? Shouldn’t we have “Limit Low R” instead?

**Comment by agriggio** [comment]
Yes, looks like a typo.. sorry!

**Comment by pbereczky** [comment]
Thank you!

**Comment by pbereczky** [comment]
I fixed typos in OpenDRT-full.txt and it includes magic strings.

The default.txt file has been extended with the keys for OpenDRT-full.ctl.
The new keys are located at the end of the default file. The first new key is CTL_ODRT_OPENDRT_FULL.

Magyar.txt is the new Hungarian language file.

Default file and Magyar file cannot have extensions. OpenDRT-full.txt file extension needs to be changed to.ctl.

[OpenDRT-full.txt](https://github.com/user-attachments/files/24255129/OpenDRT-full.txt)

[default.txt](https://github.com/user-attachments/files/24255134/default.txt)

[Magyar.txt](https://github.com/user-attachments/files/24255143/Magyar.txt)

**Comment by pbereczky** [comment]
I have attached an addendum to the translation of totm.ctl (RGB curves).
The new key is located at the end of the default file (CTL_RGB_CURVES).

[totm.txt](https://github.com/user-attachments/files/25454802/totm.txt)

[default.txt](https://github.com/user-attachments/files/25454806/default.txt)

[Magyar.txt](https://github.com/user-attachments/files/25454808/Magyar.txt)

**Comment by pbereczky** [comment]
Hi, Alberto,

Did you intentionally ignore my previous comment (February 21) or did you just forget?

**Comment by agriggio** [comment]
Sorry! I simply forgot, I don't know why :-( I will add the new translations asap.
